api-sdk/src/main/java/com/smartling/api/sdk/exceptions/ApiException.java

```diff
@@ -23,13 +30,37 @@
 {
     private static final long serialVersionUID = -397098626101615761L;
 
-    public ApiException(final String message)
+    private ApiCode apiCode;
+    private int     httpCode;
+    private List<String> messages = new ArrayList<>();
+
+    ApiException(List<String> messages, final ApiCode apiCode, int httpCode)
     {
-        super(message);
+        super(StringUtils.join(messages, " ,"));
```

Comment:
Here, you changing the meaning of exception's message. It used to be a JSON (and some clients may have used it) and now it's a comma separated string. What's the reason to do backwardly incompatible changes? I need to have a good one :)